app/models/submission.rb

```diff
@@ -9,6 +9,7 @@ class Submission < ApplicationRecord
 
   validates_uniqueness_of :fund, scope: :organisation
   validate :activity_must_be_a_fund
+  validates :deadline, date_not_in_past: true
```

Comment:
Do you think it would be a good idea to re-use our `DateWithinBoundariesValidator` here as well, just to lessen the chance of crazy future dates?